playground/nextjs-app-router/onchainkit/package.json

```diff
@@ -59,6 +60,7 @@
     "@storybook/test-runner": "^0.19.1",
     "@testing-library/jest-dom": "^6.4.6",
     "@testing-library/react": "^14.2.0",
+    "@types/qrcode": "^1",
```

Comment:
nit: these don't seem related to this PR

Reply:
yeah, they're not. they're from the QR code component we just merged. I guess when I tested things in playground they got added